proto/wallirpc.proto

```diff
@@ -10,13 +10,34 @@ service Walli {
 
 message GetInfoRequest {}
 message GetInfoResponse {
-   int32 version = 1;
-   int32 protocolversion = 2;
-   int32 blocks = 3;
-   int32 timeoffset = 4;
-   int32 connections = 5;
-   string proxy = 6;
-   int64 difficulty = 7;
-   bool testnet = 8;
-   float relayfee = 9;
+  string version = 1;
+  BtcdInfo btcdInfo = 2;
+  LndInfo lndInfo = 3;
+}
+
+message LndInfo {
+  string version = 1;
+  repeated string chains = 2;
```

Comment:
I think we should just show as little information as needed to be sure the nodes are working as intended because we will have to show that information for every chain and client we want to support. The amount of:
- blocks (for LND and BTCD to make sure LND is in sync with BTCD)
- channels
- peers (which are called `connections` in BTCD)
- and the version of the node 

should be enough but feel free to disagree and discuss other approaches.